hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDirectWriterBuilder.java

```diff
@@ -51,6 +51,7 @@ public SparkDirectWriterBuilder(JavaPairRDD<K, Either<Exception, V>> rdd){
     public DataSetWriter build() throws StandardException{
         try{
             Configuration conf=new Configuration(HConfiguration.unwrapDelegate());
+            conf.set("mapreduce.output.fileoutputformat.outputdir","/tmp"); // What should this be?
```

Comment:
Please address this comment: "What should this be?" On the service side, no sure if we can have local /tmp access. it could be s3. Can we make this configurable?

Reply:
This is an open spark 2.2 issue that we are hitting (https://issues.apache.org/jira/browse/SPARK-21549), and the work-around is explicitly set the property to a dummy value (which is valid and writable by user - say /tmp). 
Will cleanup the comment for all places.